fakes/image_test.go

```diff
@@ -3,16 +3,12 @@ package fakes_test
 import (
 	"archive/tar"
 	"fmt"
-	"io/ioutil"
-	"math/rand"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
```

Comment:
nit: these should go back into their own group

Reply:
Uhm, IntelliJ automatically formatted it in that way, I'll fix it right away